src/test/java/de/hpi/bpt/argos/api/EntityMappingEndpointTest.java

```diff
@@ -0,0 +1,355 @@
+package de.hpi.bpt.argos.api;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import de.hpi.bpt.argos.api.entityMapping.EntityMappingEndpoint;
+import de.hpi.bpt.argos.common.RestRequest;
+import de.hpi.bpt.argos.common.RestRequestFactoryImpl;
+import de.hpi.bpt.argos.core.ArgosTestParent;
+import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
+import de.hpi.bpt.argos.storage.PersistenceArtifactUpdateType;
+import de.hpi.bpt.argos.storage.dataModel.attribute.type.TypeAttribute;
+import de.hpi.bpt.argos.storage.dataModel.entity.type.EntityType;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventType;
+import de.hpi.bpt.argos.storage.dataModel.mapping.EventEntityMapping;
+import de.hpi.bpt.argos.storage.dataModel.mapping.MappingCondition;
+import de.hpi.bpt.argos.testUtil.ArgosTestUtil;
+import de.hpi.bpt.argos.testUtil.WebSocket;
+import de.hpi.bpt.argos.util.HttpStatusCodes;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+
+import static junit.framework.TestCase.assertEquals;
+
+public class EntityMappingEndpointTest extends ArgosTestParent {
+
+	private static EntityType testEntityType;
+	private static List<TypeAttribute> testEntityTypeAttributes;
+	private static EventType testEventType;
+	private static List<TypeAttribute> testEventTypeAttributes;
+
+	@BeforeClass
+	public static void initialize() {
+		testEntityType = ArgosTestUtil.createEntityType(true);
+		testEntityTypeAttributes = ArgosTestUtil.createEntityTypeAttributes(testEntityType, true);
+		testEventType = ArgosTestUtil.createEventType(false, true);
+		testEventTypeAttributes = ArgosTestUtil.createEventTypeAttributes(testEventType, true);
+	}
+
+	@Test
+	public void testCreateEntityMapping() throws Exception {
+		String targetStatus = "TargetStatus_" + ArgosTestUtil.getCurrentTimestamp();
+		int entityMappings = getMappingsCount();
+
+		WebSocket webSocket = WebSocket.buildWebSocket();
+
+		RestRequest request = RestRequestFactoryImpl.getInstance().createPostRequest(ARGOS_REST_HOST, getCreateEntityMappingUri());
+
+		JsonObject newMapping = createNewMappingJson();
+		newMapping.addProperty("TargetStatus", targetStatus);
+
+		request.setContent(serializer.toJson(newMapping));
+
+		assertEquals(HttpStatusCodes.SUCCESS, request.getResponseCode());
+
+		List<EventEntityMapping> mappings = PersistenceAdapterImpl.getInstance().getEventEntityMappingsForEventType(testEventType.getId());
+		assertEquals(entityMappings + 1, mappings.size());
+
+		EventEntityMapping createdMapping = mappings.get(mappings.size() - 1);
```

Comment:
Can you be 100% sure that the last element is the newly created one? Searching for the id could be more secure.